cpp/src/io/fst/lookup_tables.cuh

```diff
@@ -753,15 +753,15 @@ class TranslationOp {
                                              RelativeOffsetT const relative_offset,
                                              SymbolT const read_symbol) const
   {
-    return translation_op(*this, state_id, match_id, relative_offset, read_symbol);
+    return translation_op(state_id, match_id, relative_offset, read_symbol);
```

Comment:
This is just a minor improvement that I piggy-backed into this PR. The delegate that implements `translation_op` has no benefit from the extra reference of `*this`, so we removed it from the arguments.